src/sentry/integrations/slack/action_endpoint.py

```diff
@@ -39,6 +39,14 @@ class SlackActionEndpoint(Endpoint):
     authentication_classes = ()
     permission_classes = ()
 
+    def api_error(self, error):
+        logger.info('slack.action.api-error', extra=error.body)
```

Comment:
I'm assuming that `error.body` is consistently a mapping? I think I've seen logging handlers do weird things when the incorrect type is passed as `extra`. It might be safer to pass it as an item in a dictionary if you have any doubt here.

Reply:
Yeah that's a good call, that's consistent too with my dicts for slack errors (this is internal from our error, and will be an `ApiError` which does have body, but i could see it somehow ending up as a string.

Reply:
46678f1

Reply:
👍